docs/source/internals/contributing/running-tests.rst

```diff
@@ -7,6 +7,8 @@ Running tests
 
 Oscar uses pytest_ to run the tests, which can be invoked using::
 
+    $ mkvirtualenv oscar-tests
+    $ pip install -e .[test]
```

Comment:
I think we should add a sentence, saying that next commands allows to create virtual environment and install dependencies in order to run tests.